### DISCUSSION TITLE
"Aggregate" different domains in one site?

### DISCUSSION THREAD

**Discussion by MarkusAtEnreach** [plausible/analytics#1314] [Anything Else]
Hi,
we use different domains in different countries, partly as a result of M&A activities. Is it possible to use different domains in one plausible site? Administrating each URL in a single site costs a lot effort.
If this is not possible, is there an API that we can automate the administration (e.g. adding a new goal)?

**Comment by ukutaht** [comment]
> Is it possible to use different domains in one plausible site?

Not sure if this is exactly what you mean but you can set the same `data-domain` on all the sites and collect the stats into a single dashboard no problem.

**Reply by MarkusAtEnreach** [reply]
Thanks, that's solves my tension.

**Comment by LuisaAPF** [comment]
Hi,
I have a similar issue. I have 2 different sites, each one associated to a different link. My goal is to be able to see in an embedded dashboard statistics from both sites. That works using `data-domain=site1,site2`, but only if I create the embed using the link of site1 (the oldest one). If I embed using the link for site2, I only see the statistics for that site. Any ideas?

**Reply by metmarkosaric** [reply]
hi @LuisaAPF! do take a look at our instructions on how you can aggregate stats from two or more sites in the same dashboard https://plausible.io/docs/plausible-script#can-i-use-the-same-snippet-across-my-domain-and-subdomain
